.github/renovate.json5

```diff
@@ -3,6 +3,11 @@
   extends: [
     "config:recommended"
   ],
+  configMigration: true,
+  prHourlyLimit: 1,
```

Comment:
is the reason to avoid starving GHA workers?

Reply:
yeah and to make Renovate less noisy.